modules/con_notifications-configuration.adoc

```diff
@@ -0,0 +1,100 @@
+:_mod-docs-content-type: CONCEPT
+[id="notifications-configuration_{context}"]
+= NotificationsConfiguration parameter
+
+The `NotificationsConfiguration` parameter is a Kubernetes custom resource (CR) that manages notifications in a Kubernetes cluster. In {gitops-title}, the `NotificationsConfiguration` (CR) allows users to add Templates, Triggers, Services, and Subscriptions resources to an Argo CD Notifications ConfigMap.
```

Comment:
it is not a parameter.

You can just say "The `NotificationsConfiguration` is a Kubernetes Custom Resource"

Reply:
Fixed.